subsys/bluetooth/host/classic/Kconfig

```diff
@@ -109,7 +109,7 @@ config BT_L2CAP_BR_RET_TIMEOUT
 	  The Retransmission timeout shall be three times the value of flush timeout,
 	  subject to a minimum of 1000 milliseconds and maximum of 2000 milliseconds.
 
-config BT_L2CAP_BR_MONITOR_TIMEOUT
+config BT_L2CAP_RET_FC_MONITOR_TIMEOUT
```

Comment:
This Kconfig option isn't marked as experimental and has existed for a release. Should it follow the deprecation process?

At the very least, we should mention this rename in the migration guide. 

Also consider if the rename is really necessary

Reply:
> How about moving the modification of this Kconfig name to a subsequent PR if needed? @jhedberg @lylezhu2012

That's fine with me (in fact, I never requested it to be done in this PR)